client/components/tinymce/plugins/wpeditimage/plugin.js

```diff
@@ -19,6 +38,29 @@ function wpEditImage( editor ) {
 		return !! ( editor.dom.getAttrib( node, 'data-mce-placeholder' ) || editor.dom.getAttrib( node, 'data-mce-object' ) );
 	}
 
+	editor.addButton( 'wp_img_edit', {
+		tooltip: i18n.translate( 'Edit image'/*, { context: 'verb' }*/ ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **79** times:
`translate( 'Edit Image' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).